packages/openapi-v3/src/extension-point/types.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {BindingTemplate} from '@loopback/context';
+import {extensionFor} from '@loopback/core';
+import {OpenApiSpec} from '../types';
+
+/**
+ * Typically an extension point defines an interface as the contract for
+ * extensions to implement
+ */
+export interface OAISpecContributor {
```

Comment:
Maybe `OpenApiSpecEnhancer`?

Reply:
hmm....I would expect the name implies the extension will modify the spec. I kept the original name just for demo, when we agree on the proper name, I will rename all the interfaces.

Reply:
Enhancer is a term that augments an existing artifact. See https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/cglib/proxy/Enhancer.html.

I don't expect an extension to remove things from the spec, but it enhances the spec by adding additional information such as `security`.